spec/lib/requirements/topic_checker_spec.rb

```diff
@@ -25,26 +27,23 @@
       end
 
       it "returns an issue if there are no topics" do
-        issues = Requirements::TopicChecker.new(document).pre_publish_issues
+        issues = Requirements::TopicChecker.new(edition.document).pre_publish_issues
```

Comment:
Not for this PR but I wonder if we this class should take an edition as an input rather than a document.